resources/assets/js/app.js

```diff
@@ -4,7 +4,7 @@ const input = document.getElementById('url');
 form.addEventListener('submit', event => {
     event.preventDefault();
 
-    form.action = input.value.toLowerCase();
+    form.action = '/' + input.value.toLowerCase();
```

Comment:
It feels to me that this fix now just works by accident. In a month or so I'm not going to remember anymore why we prefixed `/` here.
Let's do it more explicitly by manually removing `http://` and `https://`

Reply:
Yeah, I agree that its intent is a bit vague. Alternatively, we can pull in something like https://github.com/tightenco/ziggy and use that to generate a route instead of "hacking" it in like this.